src/components/Heading/Heading.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Heading.scss';
+
+const CN = 'heading';
+
+const Heading = ({ title, position }) => (
+  <div className={`${CN}__wrapper`}>
+    <div className={`${CN}__line`} />
+    <div className={`${CN}__title ${position === 'right' ? 'right' : 'center'}`}>
+      {title}
+    </div>
+  </div>
+);
+
+Heading.propTypes = {
+  title: PropTypes.string,
+  position: PropTypes.string
```

Comment:
position: oneOf(Object.keys(posEnum))

what do you think?

Reply:
great solution, thank you!